docs/external-ip-and-loadbalancer-ingress-in-shared-gateway-mode.md

```diff
@@ -0,0 +1,45 @@
+
+OVN Kubernetes in Shared Gateway mode implements both External IPs and LoadBalancer Ingress IPs (`service.Status.LoadBalancer.Ingress`) in the form of OVN load balancers. These OVN load balancers live on all of the Kubernetes nodes and are thus highly available and ready for load sharing. A consequence of this highly available design is that OVN Kubernetes makes it the administrator's responsibility to route traffic to the Kubernetes nodes for both of these VIP types. 
+
+In an environment where External IPs and LoadBalancer Ingress VIPs happen to be part of the nodes' subnets, administrators might expect that OVN Kubernetes answer to ARP requests to these VIPs. However, this is not the case. The administrator is responsible for routing packets to the Kubernetes nodes and they cannot rely on the network plugin for this to happen.
+
+#### External IP
+
+The Kubernetes documentation states that External IPs are to be handled by the cluster administrator and are not the responsibility of the network plugin. See [https://kubernetes.io/docs/concepts/services-networking/service/#external-ips](https://kubernetes.io/docs/concepts/services-networking/service/#external-ips):
+~~~
+External IPs
+If there are external IPs that route to one or more cluster nodes, Kubernetes Services can be exposed on those externalIPs. Traffic that ingresses into the cluster with the external IP (as destination IP), on the Service port, will be routed to one of the Service endpoints. externalIPs are not managed by Kubernetes and are the responsibility of the cluster administrator.
+~~~
+
+#### LoadBalancer Ingress (service.Status.LoadBalancer.Ingress)
+
+For a service's Status LoadBalancer Ingress field `service.Status.LoadBalancer.Ingress`, the aforementioned statement applies in exactly the same manner. Both External IP and `service.Status.LoadBalancer.Ingress` should behave the same from the network plugin's behavior, and it is the administrator's responsibility to get traffic for the VIPs into the cluster. 
+
+#### Implementation details
+
+OVN Kubernetes in Shared Gateway mode exposes External IPs and `service.Status.LoadBalancer.Ingress` VIPs as OVN load balancers on every node in the cluster. However, OVN Kubernetes will not answer to ARP requests to these VIP types, even if they reside on a node local subnet. This is because otherwise, every node in the cluster would answer with its own ARP reply to the same ARP request, leading to potential issues with stateful network flows that are tracked by conntrack. See the discussion in [https://github.com/ovn-org/ovn-kubernetes/issues/2407](https://github.com/ovn-org/ovn-kubernetes/issues/2407) for further details.
+
+In fact, OVN Kubernetes implements bypass rules for ARP requests to these VIP types on the external bridge (`br-ex` or `breth0` in most deployments). Any ARP request to such an IP that comes in from the physical port will bypass the OVN dataplane on purpose. 
```

Comment:
```suggestion
In fact, OVN Kubernetes implements bypass rules for ARP requests to these VIP types on the external bridge (`br-ex` or `breth0` in most deployments). Any ARP request to such an IP that comes in from the physical port will bypass the OVN dataplane and be sent to host's networking stack on purpose. 
```
Should we say something like `So if an ARP reponse to these VIP's are expeced, make sure they are configured as an interface on the host?`

Reply:
~~~
In fact, OVN Kubernetes implements explicit bypass rules for ARP requests to these VIP types on the external bridge (`br-ex` or `breth0` in most deployments). Any ARP request to such an IP that comes in from the physical port will bypass the OVN dataplane and be sent to host's networking stack on purpose. If an ARP reponse to a VIP is expeced, make sure the VIP is added to the host's networking stack.
~~~